content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -43,11 +43,7 @@ Note that if your `ca-cert.pem` is the same as `root-cert.pem`, the `cert-chain.
 These files are ready to use in the `samples/certs/` directory.
 
   {{< tip >}}
-  The default Citadel installation mounts the `cacert` secret as a volume under `/etc/certs`.
-  It is configured via [command line options](/docs/reference/commands/istio_ca/index.html)
-  to load certificates and keys from the predefined file names used in the command below
-  (i.e., root certificate must be in a file named `root-cert.pem`, Citadel key in
-  `ca-key.pem`, etc.)
+  The default Citadel installation sets [command line options](/docs/reference/commands/istio_ca/index.html) to configure the location of certificates and keys based on the predefined secret and file names used in the command below (i.e., secret named `cacert`,root certificate in a file named `root-cert.pem`, Citadel key in `ca-key.pem`, etc.)
```

Comment:
missing space (" ") character after comma: `cacert`,root certificate ...

Also one more thought, maybe it would be good to add another sentence, something like this:

You must use these specific file names for your certificates and keys unless you reconfigure Citadel when you deploy it.

Reply:
updated: `You must use these specific secret and file names, or reconfigure Citadel when you deploy it.`